zebra/zapi_msg.c

```diff
@@ -1782,6 +1869,21 @@ static void zread_route_add(ZAPI_HANDLER_ARGS)
 			bnhg->nhe->nhg.nexthop = nexthop;
 
 		last_nh = nexthop;
+
+		if (CHECK_FLAG(api.flags, ZEBRA_FLAG_SEG6LOCAL_ROUTE)
```

Comment:
I don't understand this change: a) this is processing backup nexthops - is SEG6LOCAL info only relevant for backup nexthops? b) the `add_sorted` below can't be correct - we've already enqeued this backup nexthop on the correct list just above; we don't want to enqueue it twice?

Reply:
Aaaa. I see. It's my mistake... I'll just fix it

Reply:
a)

- SEG6LOCAL is relevant both normal-nexthop and backup-nexthop like-a labels.

b)

- Yes, this nexthop_grou_add_sorted is my miss-coding. I'm sorry. :bow:

Reply:
I fixed it.